docs/usage.rst

```diff
@@ -195,6 +195,9 @@ This is done on a per-course basis and can be modified in 3 different locations
 
     .. image:: images/other_course_settings_data.png
 
+    You can skip the course key
```

Comment:
This should be the preferred way. Can you put it on top of the other.

Then you can say, "You could optionally use the course key as well" ...

Reply:
done